store/sqlstore/team_store.go

```diff
@@ -194,41 +194,37 @@ func (s SqlTeamStore) Save(team *model.Team) store.StoreChannel {
 	})
 }
 
-func (s SqlTeamStore) Update(team *model.Team) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		team.PreUpdate()
+func (s SqlTeamStore) Update(team *model.Team) *model.AppError {
```

Comment:
I would expect to maintain the previous behavior here, which is returning the updated team plus the error. This is consistent with other store Update calls in other stores like `User.Update` or `Channel.Update`.

Reply:
Updated.